### PR TITLE
Refactored tray components into new Tray component

### PR DESCRIPTION
**Pull request by rladdusaw**

### CODE REVIEW

File: src/components/metadata/FindingaidesMetadata.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<template>
```
Comment (sandbergja) [inline src/components/metadata/FindingaidesMetadata.vue:1]: Could we rename this component to FindingaidsMetadata? (no e in findingaids)?

File: src/App.vue [hunk 2]
```diff
@@ -1,12 +1,8 @@
 <script setup lang="ts">
```
Comment (sandbergja) [inline src/App.vue:1]: Thanks for simplifying this component!

File: src/config/ScopeTitleMap.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export default {
```
Comment (sandbergja) [inline src/config/ScopeTitleMap.ts:1]: Nice, I think having this mapping in its own object makes things clear.

**Review by sandbergja — CHANGES_REQUESTED** [review]
Looks great, thanks @rladdusaw !  Just one typo caught my eye

File: src/components/metadata/FindingaidesMetadata.vue (hunk 1 above)
Comment (rladdusaw) [reply src/components/metadata/FindingaidesMetadata.vue]: Thanks for catching that!

**Review by sandbergja — APPROVED** [review]
🐈‍⬛

**Pull request merged**